Kentico/kontent-starter-corporate-next-js
# Rewrite data fetching approach

### Motivation

Related to #37

* Pass just raw object via Next.js static props and recreate the JS object on a client
* Removed Material UI
* Remove forgotten importing of the whole `lodash`

### Checklist


### How to test

Click through the app if everything works.
Try preview (secret is `netlify_secret`)
Check readme if it is clear.
